doc/gallery-src/framework/run_grids2_cartesian.py

```diff
@@ -0,0 +1,33 @@
+"""
+Make a Cartesian grid
+=====================
+
+This builds a Cartesian grid with squares 1 cm square. with the z coordinates
+provided explicitly. It is also offset in 3D space to X,Y,Z = 10, 5, 5 cm. 
+
+Learn more about :py:mod:`grids <armi.reactor.grids>`.
+"""
+import itertools
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
```

Comment:
Unused?

Reply:
actually it is secretly used.  By importing it it activates the `3d` projection option. If you take it out it fails!